### PR TITLE
V0.90 Update dict helper methods to retrieve and incorporate computed-model metadata

### PR DESCRIPTION
**Pull request by piehld**

### CODE REVIEW

**Review by brindakv — CHANGES_REQUESTED** [review]
@piehld Please see my comments. 

I also suggest to add the header updates to `DictMethodCompModelHelper`, `DictMethodEntityHelper`, `DictMethodEntityInstanceHelper`, and `DictMethodResourceProvider`. 

I wonder why no update was required for populating `entity_instance_container_identifiers`.

File: rcsb/utils/dictionary/DictMethodCompModelHelper.py [hunk 1]
```diff
@@ -280,4 +281,61 @@ def consolidateGlobalQAScores(self, dataContainer, catName, **kwargs):
...
+        Returns:
+            bool: True for success or False otherwise
+
+        For example:
+
+        loop_
+        _rcsb_comp_model_provenance.source_filename
+        _rcsb_comp_model_provenance.source_url
+        _rcsb_comp_model_provenance.source_db
+        _rcsb_comp_model_provenance.source_id
```
[the remark this answers is not in the document]
Comment (brindakv) [reply rcsb/utils/dictionary/DictMethodCompModelHelper.py:302]: I am referring to line 302 `_rcsb_comp_model_provenance.source_id`

File: rcsb/utils/dictionary/DictMethodCompModelHelper.py [hunk 2]
```diff
@@ -280,4 +281,61 @@ def consolidateGlobalQAScores(self, dataContainer, catName, **kwargs):
...
+            entryId = tObj.getValue("id", 0)
+
+            compModelId = self.__mcP.getInternalCompModelId(entryId)
+            if not compModelId:
+                logger.error("Unable to map computed-model entryId/sourceId (%s) to internal identifier - sourceId key not found.", entryId)
+                return False
+
+            compModelD = self.__mcP.getCompModelData(compModelId)
+
+            # cObj.setValue(entryId, "source_id", 0)
```
[the remark this answers is not in the document]
Comment (piehld) [reply rcsb/utils/dictionary/DictMethodCompModelHelper.py]: Thanks @brindakv . Done.

File: rcsb/utils/dictionary/DictMethodCompModelHelper.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (piehld) [reply rcsb/utils/dictionary/DictMethodCompModelHelper.py]: Done.

**Review by brindakv — APPROVED** [review]
@piehld Thanks. Looks good.

**Pull request merged**